### DISCUSSION TITLE
IfcConvert to render a large curved wall

### DISCUSSION THREAD

**Discussion by DaiNguyenTM** [IfcOpenShell/IfcOpenShell#2636] [Q&A] [answered]
Hi all,
I ran into a problem while using IfcConvert to generate svg.
IfcConvert can't render a large curved wall, so all walls/spaces...have an impact on that.

![image](https://user-images.githubusercontent.com/119857165/205923433-f6aad183-cc6a-4297-b322-aab040d2404b.png)

Can you please help ?

Many thanks.

**Comment by Moult** [comment, marked as answer]
Is there any difference between this and the bug report you submitted here? https://github.com/IfcOpenShell/IfcOpenShell/issues/2629

**Reply by DaiNguyenTM** [reply]
No, it is the same.  I'd like to use IfcConvert to replace the our old converter wrote in C#.
Thank you for your helps

**Reply by Moult** [reply]
In that case I recommend you wait until perhaps @aothms gets a chance to look at your bugreport, cross posting here won't help :)
